api/src/main/java/openconsensus/trace/data/Event.java

```diff
@@ -30,54 +30,54 @@
  */
 @Immutable
 @AutoValue
-public abstract class Annotation {
+public abstract class Event {
   private static final Map<String, AttributeValue> EMPTY_ATTRIBUTES =
       Collections.unmodifiableMap(Collections.<String, AttributeValue>emptyMap());
 
   /**
-   * Returns a new {@code Annotation} with the given description.
+   * Returns a new {@code Event} with the given name.
    *
-   * @param description the text description of the {@code Annotation}.
-   * @return a new {@code Annotation} with the given description.
-   * @throws NullPointerException if {@code description} is {@code null}.
+   * @param name the text name of the {@code Event}.
+   * @return a new {@code Event} with the given name.
+   * @throws NullPointerException if {@code name} is {@code null}.
    * @since 0.1.0
    */
-  public static Annotation fromDescription(String description) {
-    return new AutoValue_Annotation(description, EMPTY_ATTRIBUTES);
+  public static Event fromDescription(String name) {
+    return new AutoValue_Event(name, EMPTY_ATTRIBUTES);
```

Comment:
Where is AutoValue_Event ?  I am not seeing it in master, nor defined in this pr.

Reply:
https://github.com/google/auto/blob/master/value/userguide/index.md